app/modules/settings/components/account.svelte

```diff
@@ -151,7 +166,17 @@
         on:click={toggleFediversable} />
       {i18n('Fediverse integration')}
     </label>
-    <Flash bind:state={flashFediversable} />
+    <!-- I18n copy/pasted from fediversable_description last paragraph: To prevent sending too many messages, when several books are added at about the same time, only one summary message will be sent. -->
+    <p class="note">{@html I18n('pooling_activities_description')}</p>
```

Comment:
We don't need underscored i18n keys anymore
```suggestion
    <p class="note">{i18n('To prevent sending too many messages, when several books are added at about the same time, only one summary message will be sent.')}</p>
```